.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,17 @@
-What does it do?
+### What does it do?
 
-What important points reviewers should know?
+### What important points reviewers should know?
 
-Is there something left for follow-up PRs?
+### Is there something left for follow-up PRs?
 
-What alternative implementations were considered?
+### What alternative implementations were considered?
 
-Are there relevant PRs or issues in other repositories (Substrate, Polkadot, Frontier, Cumulus)?
+### Are there relevant PRs or issues in other repositories (Substrate, Polkadot, Frontier, Cumulus)?
 
-[ ] Does it require a purge of the network?
-[ ] You bumped the runtime version if there are breaking changes in the **runtime** ?
-[ ] Does it require changes in documentation/tutorials ?
+### What value does it bring to the blockchain users?
 
-What value does it bring to the blockchain users?
+## Checklist
```

Comment:
```suggestion
### Checklist
```

To match the other headings? I don't fell strongly about it, just mentioned it in case it was an accident.

Reply:
Was on purpose to differentiate :)